core/src/main/java/apoc/coll/Coll.java

```diff
@@ -149,6 +149,7 @@ public Double avg(@Name("coll") List<Number> list) {
         }
         return (avg/(double)list.size());
     }
+    @NotThreadSafe
```

Comment:
Are we marking this not thread safe because it tries to execute with `runtime=slotted`?